cluster/images/provider-aws/Dockerfile

```diff
@@ -43,7 +43,7 @@ ENV TERRAFORM_VERSION ${TERRAFORM_VERSION}
 ENV TERRAFORM_PROVIDER_SOURCE ${TERRAFORM_PROVIDER_SOURCE}
 ENV TERRAFORM_PROVIDER_VERSION ${TERRAFORM_PROVIDER_VERSION}
 # ENV TERRAFORM_NATIVE_PROVIDER_PATH ${PLUGIN_DIR}/${TERRAFORM_PROVIDER_DOWNLOAD_NAME}_v${TERRAFORM_PROVIDER_VERSION}_x5
-ENV TF_APPEND_USER_AGENT "upbound-provider-aws/${CROSSPLANE_PROVIDER_VERSION} universal-crossplane"
+ENV TF_APPEND_USER_AGENT "crossplane-provider-aws-upbound/${CROSSPLANE_PROVIDER_VERSION}"
```

Comment:
Just getting to this. So my thinking was that we need to tell two things: it is a Crossplane provider and it is the one under upbound with a given version. And we should tell the version in the one where we tell where to find it only, i.e. `upbound-provider-aws`. For example, we're [using](https://github.com/crossplane-contrib/provider-aws/blob/38a0d38dbd8f428955d053472ef27ff65ef6ee13/pkg/clients/aws.go#L88) `crossplane-provider-aws/version` in the classic community provider, too, as far as I can tell, so we'll end up with a conflict when doing the filtering.

I'd like to open another change if `crossplane-provider upbound-provider-aws/${CROSSPLANE_PROVIDER_VERSION}` looks good to you from AWS perspective.

Reply:
Good point. Ideally, I'd like to have an unique short string for each provider like I originally had. I think it's easier to identify in CloudTrail and aggregate for analysis for our customers. Useragent strings typically don't contain spaces in product names and repeating the word provider seems weird. What do you think about `crossplane-provider-upbound-aws/${VERSION}`?
W/e we choose, I think it's a good idea to be consistent across providers (aws or not). I imagine in multi-cloud settings, it would make analysis easier.

Reply:
> I'd like to have an unique short string for each provider like I originally had

Agreed. I think the Github path provides that, i.e. `crossplane-contrib/provider-aws` -> `crossplane-contrib-provider-aws` as well as `upbound/provider-aws` -> `upbound-provider-aws`.

> Useragent strings typically don't contain spaces in product names and repeating the word provider seems weird.

I thought the space was actually how to add more labels. Are you saying the whole user-agent string is one opaque string in CloudTrail where you can't really analyze if there was space? (btw just `crossplane` would work as well as long as it's good to differentiate from TF or other tools that use _provider_ term)

Reply:
I think it's important to have or preferably lead with `crossplane` in the useragent product name. `upbound-provider-aws` doesn't really tell me what it is. 

Usually, useragents are formatted like `${PRODUCTNAME}/${VERSION} (${COMMENT})`. I think it's specified in a RFC and you can't have spaces in `${PRODUCTNAME}`.  I don't think it's enforced, but it is defined and many useragents follow that convention. 
You can certainly analyze in CloudTrail or w/e analysis tools with spaces, but becomes a little more complicated.